admin_ui/backend/api/config.py

```
@@ -665,12 +698,38 @@ def replace(match):
+                        return data
+
+                try:
+                    data = await _try_connect(ws_url)
+                    effective_url = ws_url
+                except Exception as e:
+                    alt = _fallback_ws_url(ws_url)
+                    if alt != ws_url:
+                        data = await _try_connect(alt)
+                        effective_url = alt
+                    else:
+                        raise e
```

Ruff (0.14.10)

Use `raise` without specifying exception name

Remove exception name

(TRY201)